emperor/support_files/js/util.js

```diff
@@ -89,6 +89,27 @@ define(['underscore'], function(_) {
 
   /**
    *
+   * Split list of string values into numeric and non-numeric values
+   * @param {String[]} values The values to check
+   * @return {Numeric[]} Array of numeric values, converted to numbers
+   * @return {String[]} Array of non-numeric values, still as strings
+   */
+   function splitNumericValues(values) {
+    var numeric = [];
+    var nonNumeric = [];
+    _.each(values, function(element) {
+        if (!isNaN(element) && isFinite(+element) && Number(element) !== 0) {
+          numeric.push(+element);
+        }
+        else {
+          nonNumeric.push(element);
+        }
+      });
+    return [numeric, nonNumeric];
```

Comment:
I was thinking about how this function is being used and it may make more sense to return an object instead of the array? Something like:

```javascript
function thyFunction(values){
    // ...
    return {numeric: numeric, nonNumeric: nonNumeric};
}
```

Later the code that uses this can just refer to the appropriate properties in a named manner instead of having to assign these to variables, what do you think?